custom/enikshay/integrations/nikshay/repeater_generator.py

```diff
@@ -86,6 +88,12 @@ def _base_properties(self, repeat_record):
             "IP_FROM": server_ip,
         }
 
+    def use_new_2b_app_structure(self, person_case, episode_case):
```

Comment:
sorry i don't completely understand this. What is V2, V1?

also sheel said we would eventually remove the logic for 1.1 and just keep for 2.0 after a month since everyone would move to that so okay to keep things simple.